go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/decred/dcrdata/v4
 
+replace github.com/decred/dcrdata/gov/agendas => ./gov/agendas
```

Comment:
The change to agendas.VoteTracker is a hack just for lite mode.  Further, 4.1 is about to become legacy, so we aren't going to bother with a new gov/agendas module tag.